tools/vscode-azurewebpubsub/src/utils.ts

```diff
@@ -42,3 +44,16 @@ export function isUrlValid(url: string): boolean {
         return false;
     }
 }
+
+export async function loadPackageInfo(context: ExtensionContext) {
+    const raw = await fs.promises.readFile(context.asAbsolutePath("./package.json"), { encoding: 'utf-8' });
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+    const { publisher, name, version, aiKey } = JSON.parse(raw);
```

Comment:
give it a default value or directly throw instead of `as string` to defend a bad `package.json`, maybe add some test to avoid a bad package.json